css/style.css

```diff
@@ -25,7 +34,31 @@ li {
   font-weight: bold;
 }
 
-.hero h1 {
+.upper {
+  text-transform: uppercase;
+}
+
+.cap {
+  text-transform: capitalize;
+}
+
+.black {
+  color: var(--black, #172b4d);
+}
+
+.gray {
+  color: var(--gray, #7a869a);
+}
+
+.highlight {
+  color: var(--black-light, #344563);
+}
```

Comment:
- [ ] Please, make sure that the `.highlight` class is used multiple times in the `index.html` file first.